litezip/completezip.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+
+from .main import parse_litezip
+
+
+__all__ = (
+    'convert_completezip',
+)
+
+
+def convert_completezip(path):
+    """Converts a completezip file structure to a litezip file structure.
+    Returns a litezip data structure.
+
+    """
+    for filepath in path.glob('**/index_auto_generated.cnxml'):
```

Comment:
I checked w/ the example completezip you pointed Alan at below. Unfortunately the autogen files are not bit identical to the index.cnxml. They differ in having an explicit version instead of latest in the metadata, and the timezone format is numeric instead of symbolic. In one case, keywords are in a different order. I'll stand by if `document[@cnxml-version=0.7]` matches in index.cnxml, we toss autogen, but if it's older, we rename autogen to index.cnxml.